src/holofuel/components/SideMenu/SideMenu.js

```diff
@@ -5,9 +5,8 @@ import { Link, useLocation } from 'react-router-dom'
 import HashAvatar from 'components/HashAvatar'
 import { presentAgentId } from 'utils'
 import CopyAgentId from 'holofuel/components/CopyAgentId'
-import Button from 'components/UIButton'
-import Loading from 'components/Loading'
 import BackIcon from 'components/icons/BackIcon'
+import MenuButton from 'holofuel/components/MenuButton'
```

Comment:
In Firefox, the menuButton has a bit of an alignment bug.  Here's a screenshot for visual clarity:
![image](https://user-images.githubusercontent.com/30459942/88362591-ac459b80-cd42-11ea-9327-fbe3b67ac8b3.png)

Reply:
good catch